.github/workflows/run_tests.yaml

```diff
@@ -19,7 +19,15 @@ jobs:
         run: docker-compose build vuln4shift_unit_tests
 
       - name: Run unit tests
-        run: docker-compose run --rm vuln4shift_unit_tests
```

Comment:
If there is a local dir mount, the `--rm` can stay here.

Reply:
But it doesn't really matter, the container will be removed once the GH action ends anyway.